db/migrate/20230316050655_create_yardsale_tags.rb

```diff
@@ -0,0 +1,10 @@
+class CreateYardsaleTags < ActiveRecord::Migration[7.0]
+  def change
+    create_table :yardsale_tags do |t|
+      t.integer :tag_id
+      t.integer :yardsale_id
```

Comment:
I think these should be foraging keys. Also, we may not need this table if we end up going with the act_as_taggable gem

Reply:
can i just delete this migration file?